src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/AudioPlayer/AudioPlayerContent/AudioPlayerContent.tsx

```diff
@@ -0,0 +1,93 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box } from '@mui/material';
+
+import { useCustomFormContext } from 'modules/Builder/hooks';
+import { StyledTitleMedium, StyledBodyErrorText, theme, variables } from 'shared/styles';
+import { MLPlayer } from 'modules/Builder/components/MLPlayer';
+import { MediaType } from 'modules/Builder/components';
+
+import { AddAudio } from '../AddAudio';
+import { UploadAudio } from '../UploadAudio';
+import { RecordAudio } from '../RecordAudio';
+import { RemoveAudioPopup } from '../RemoveAudioPopup';
+
+export const AudioPlayerContent = ({
+  media,
+  setMedia,
+  name,
+}: {
```

Comment:
Pls move type to AudioPlayerContent.types.ts

Reply:
done